src/native/configuration/globalconfig.rs

```diff
@@ -113,6 +113,7 @@ impl GlobalConfig {
     }
 
     /// Save the Client data to the config
+    #[allow(unused)]
     pub async fn save_client(&mut self, client: Client) -> Result<(), NativeError> {
```

Comment:
the client should not populate the config

the config should populate the client